OPERATOR.rst

```diff
@@ -191,21 +198,18 @@ To do a promotion:
 
        git push github
 
-#. If a reindex is necessary:
-
-   * Reindexing ``prod`` currently require manual assistance from the lead to
-     increase slot commitment and indexer concurrency. Coordinate with the lead
-     or remove this paragraph if this is no longer necessary.
-
-   * To prevent prod from going down longer than necessary, preemptively cancel
-     the integration test before it runs, and run ``early_reindex``.
+#. If a reindex is necessary, preemptively cancel the integration test before it
```

Comment:
`If a early reindex is necessary` ? Or are we always using early-reindexs now for prod?

Reply:
We are always using early reindexes. This isn't new information, I'm just consolidating some of what was already there now that we don't need to ask Hannes to reserve slots, etc.

Reply:
@jessebrennan 'always let the IT run, then decide if you need to revert or reindex'